app/workers/push_notifications/cleanup_worker.rb

```diff
@@ -0,0 +1,21 @@
+module PushNotifications
+  class CleanupWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :low_priority, retry: 10
+
+    def perform
+      redis = Redis.new(url: ENV["REDIS_RPUSH_URL"])
+      cursor = 0
+      until (cursor, keys = redis.scan(cursor)).first.to_i.zero?
```

Comment:
This is something @citizen428 mentioned up the thread that probably got lost in the wash.

```suggestion
      until (cursor, keys = redis.scan(cursor, match: "rpush:notifications:*")).first.to_i.zero?
```

Reply:
Oof, I did miss that, thanks!